src/lib/structures/Finalizer.js

```diff
@@ -18,7 +18,7 @@ class Finalizer {
 	/**
 	 * @param {KlasaClient} client The Klasa Client
 	 * @param {string} dir The path to the core or user finalizer pieces folder
-	 * @param {Array} file The path from the pieces folder to the finalizer file
+	 * @param {string[]} file The path from the pieces folder to the finalizer file
```

Comment:
should just be a string here.

Reply:
You're right, in the typings, I wrote

```typescript
public file: string;
```

But wrote the wrong type in the JSDocs (got confused with Command). Fixed 👍